erpnext/telephony/doctype/call_log/call_log.py

```diff
@@ -33,6 +33,18 @@ def before_insert(self):
 		if lead:
 			self.add_link(link_type='Lead', link_name=lead)
 
+		# Add Employee Name
+		if self.is_incoming_call():
+			# Taking the last 10 digits of the number
+			emp_number_reversed = (self.get("to"))[-1:-11:-1]
+			emp_number = emp_number_reversed[-1::-1]
+			employee = frappe.get_all("Employee", filters={
+				"cell_number": ["like", "%"+emp_number+"%"]
+			}, fields=["first_name", "middle_name", "last_name", "user_id"])
+
+			self.call_received_by = get_employee_name(employee[0])
```

Comment:
Let's keep `call_rececived_by` a link field. We can also, enable the [title for link](https://frappeframework.com/docs/v13/user/en/basics/doctypes/form_&_view_settings#title-field) for Employee doctype.